news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+After an arduous and exciting five weeks of matches in which more than 700 initial contestants were narrowed down to a mere 26, the HappyStick Summer Tour reached its climax Sunday, August 13 with the Grand Final matches. Over the course of the tournament, 768 players from [x] countries across 13 rank brackets ranging from the 250 - 500 to 100,000+ tiers fought their way through a grand total of 537 matches from Group Stages through Bracket Stages until only two players remained in each. Those select few fought it out mano a mano on the grand stage of the finals where they clashed for the crown of Summer Tour Champion.
```

Comment:
need to shorten the entry paragraph a lot. contains a lot of good information but also a lot of superfluous stuff. entry/first paragraphs are shown on the homepage and gotta have a "hook" to them that is short and sweet (4 line paragraph max) which lures people into reading the rest of the article